src/helpers/tools.ts

```diff
@@ -1,2 +1,18 @@
 // eslint-disable-next-line no-promise-executor-return
 export const wait = (delay: number) => new Promise(resolve => setTimeout(resolve, delay));
+
+export const getElementVisibleChildrenIndices = (element: Element): { start: number, end: number } => {
```

Comment:
The function looks fine, but is it being used anywhere in the app?

Reply:
Nope. Removed!